src/earn/poolInfo.ts

```diff
@@ -32,3 +36,47 @@ export async function fetchAavePoolInfo(assetAddress: Address) {
     throw err
   }
 }
+
+/** @beta - Exclude from Knip dep check */
```

Comment:
is this required?

Reply:
Removed in 3c133d6c3651dd08098da0fccb348aec1821ed50!